openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -310,6 +341,17 @@ class CoursewareInformation(RetrieveAPIView):
         * name: Name of the course
         * number: Catalog number of the course
         * org: Name of the organization that owns the course
+        * related_programs: A list of objects that contains related program data including:
```

Comment:
Nit: say something about about how it's the programs related to the given course or something?